src/plugins/region_map/server/plugin.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  PluginInitializerContext,
+  CoreSetup,
+  CoreStart,
+  Plugin,
+  Logger,
+} from 'opensearch-dashboards/server';
+
+import { OpenSearchService } from './services';
+import { opensearch } from '../server/routes';
+import { getUiSettings } from './ui_settings';
+
+export class RegionMapPlugin implements Plugin<object, object> {
+  private readonly logger: Logger;
+
+  constructor(initializerContext: PluginInitializerContext) {
+    this.logger = initializerContext.logger.get();
+  }
+
+  public setup(core: CoreSetup) {
+    this.logger.debug('RegionMap: Setup');
+    const opensearchClient = core.opensearch.legacy.createClient('opensearch');
+    const opensearchService = new OpenSearchService(opensearchClient);
+    const router = core.http.createRouter();
+    core.uiSettings.register(getUiSettings());
+    opensearch(opensearchService, router);
+  }
+
+  public start(core: CoreStart) {
```

Comment:
In the js file this didnt have any param, was there any logic that needs to be updated to include param.

Reply:
nope! there was nothing within this block

Reply:
style nit for unused param:

```suggestion
  public start(_core: CoreStart) {
```